apps/ejabberd/src/mongoose_commands.erl

```diff
@@ -115,6 +119,7 @@
           function :: atom(),                         %% function to call
           action :: action(),                         %% so that the HTTP side can decide which verb to require
```

Comment:
According to **Elvis**:
> Line 120 is too long:           action :: action(),                         %% so that the HTTP side can decide which verb to require.